src/popup/components/TransactionDetailsBase.vue

```diff
@@ -261,49 +167,32 @@ export default defineComponent({
   components: {
     PayloadDetails,
     TransactionErrorStatus,
-    TransactionDetailsPoolTokens,
-    TransactionTokens,
     TransactionOverview,
     TokenAmount,
     DetailsItem,
     LinkButton,
     CopyText,
-    SwapRoute,
-    SwapRates,
     AnimatedPending,
     ExternalLink,
-    Avatar,
-    DialogBox,
   },
   props: {
     transaction: { type: Object as PropType<ITransaction | undefined>, required: true },
     coinSymbol: { type: String, required: true },
     transactionFee: { type: Number, required: true },
-    isSwap: { type: Boolean },
-    isPool: { type: Boolean },
     tokenSymbol: { type: String, required: true },
     totalAmount: { type: Number, required: true },
-    isErrorTransaction: { type: Boolean },
-    isDexAllowance: { type: Boolean },
-    isDex: { type: Boolean },
+    isErrorTransaction: Boolean,
+    showHeader: Boolean,
+    hideAmount: Boolean,
```

Comment:
Can you please move the boolean props to the end of props list? That's a common pattern within the industry.